src/controllers/BaseController.ts

```diff
@@ -23,18 +24,18 @@ const createChangeModeAwareNavigationProxy = (step: Navigation): Navigation => {
                 if (req.query.cm === '1') {
                     return CHECK_YOUR_APPEAL_PAGE_URI;
                 }
-                return (target[propertyName] as (req: Request) => string).apply(this, [req])
-            }
+                return (target[propertyName] as (req: Request) => string).apply(this, [req]);
+            };
         }
-    })
+    });
 };
 
 export abstract class BaseController<FORM> extends BaseAsyncHttpController {
     protected constructor(@unmanaged() readonly template: string,
-                          @unmanaged() readonly navigation: Navigation,
-                          @unmanaged() readonly formSchema?: AnySchema,
-                          @unmanaged() readonly formSanitizeFunction?: FormSanitizeFunction<FORM>,
-                          @unmanaged() readonly formSubmissionProcessors?: FormSubmissionProcessorConstructor[]) {
+        @unmanaged() readonly navigation: Navigation,
```

Comment:
That change seems not needed, I suggest revert. Previous formatting read better.

Reply:
Ooops this one was accident